public/signup.html

```diff
@@ -1,4 +1,3 @@
-<!-->
```

Comment:
dont ever use inner style! (style attribute, like in line 16) unless you have a very good reason

Reply:
why there is an empty div at the end of the file?